worlds/minecraft/docs/en_Minecraft.md

```diff
@@ -9,7 +9,7 @@ config file.
 
 Recipes are removed from the crafting book and shuffled into the item pool. It can also optionally change which
 structures appear in each dimension. Crafting recipes are re-learned when they are received from other players as item
-checks, and occasionally when completing your own achievements.
+checks, and occasionally when completing your own achievements. See below for what recipes are shuffled.
```

Comment:
```suggestion
checks, and occasionally when completing your own achievements. See below for which recipes are shuffled.
```
I *think* which is proper here grammaticaly seeking. Unless someone with more grammar knowledge knows says otherwise.